src/sugar3/speech.py

```diff
@@ -220,6 +222,28 @@ def __init__(self, **kwargs):
         self._save_timeout_id = -1
         self.restore()
 
+    def _marked_messages_cb(self, player, value):
+        '''
+        This method provides access to mark message values which can be used
```

Comment:
This is great documentation, but it is in the wrong place.  The method is not to be called by the activity.  The method is private to the SpeechManager class.

Reply:
Yes the method is private to the SpeechManager class, the documentation was basically to show how to connect the signal, the necessary arguments. I can remove it.

Reply:
@quozl waiting for your opinion here.

Reply:
I can't say no to documentation, but I will say no to inconsistent documentation.  It would be great to have automatically generated documentation for this module.  But it need not happen in this pull request, because the module at the moment has almost no documentation.